spec/shared_examples_for_update_metadata_source_action.rb

```diff
@@ -94,8 +95,9 @@
       file_2_path = File.join(dir, '2.txt')
       File.write(file_2_path, 'value 2')
 
-      described_class.run(
+      run_described_action(
         po_file_path: output_path,
+        release_version: '1.0',
```

Comment:
@AliSoftware case in point for using this approach over `run`. It highlighted the fact that `release_version` is a required `ConfigItem`, even though not passing it to the `run` call doesn't result in an error 👍

Reply:
Ah! Interesting!

As a side note (and I think we have this tracked somewhere, just not sure to remember where) one of the improvements we might want to implement on the replacement for those actions is to auto-detect the release_version an stop having to pass it to the actions. Given the action should already have everything it needs to guess/determine the current release version (either by reading it from the xcconfig file or by the release/* branch name), it would be nice to remove the need for it during your future refactoring 🙂